app/script/entity/Conversation.js

```diff
@@ -394,7 +394,9 @@ z.entity.Conversation = class Conversation {
     return this.unreadEvents()
       .slice()
       .reverse()
-      .find(messageEntity => messageEntity.visible() && messageEntity.is_content() && messageEntity.isSelfMentioned());
+      .find(messageEntity => {
+        return messageEntity.visible() && messageEntity.is_content() && messageEntity.isSelfMentioned(this.self.id);
```

Comment:
@gregor Unfortunately, there is not much we can do about this line break. 
The previous version was already 2 characters away from breaking and I don't think we want to rename our convention `messageEntity` for the sake of formatting.
Any other ideas?